css/styles.css

```
@@ -542,7 +577,7 @@ input:focus {
         border-color: #5DADE2;
     }
     td p {
-        font-size:3rem;
+        font-size:3em;
```

Agrega un espacio después de los dos puntos.
```suggestion
        font-size: 3em;
```